curriculum/challenges/english/25-front-end-development/lecture-html-fundamentals/670838e914096b194b0c51aa.md

```diff
@@ -0,0 +1,121 @@
+---
+id: 670838e914096b194b0c51aa
+title: What Is an HTML Boilerplate, and Why Is It Important?
+challengeType: 11
+videoId: nVAaxZ34khk
+dashedName: what-is-an-html-boilerplate
+---
+
+# --description--
+
+Watch the video and answer the questions below.
+
+# --questions--
+
+## --text--
+
+Where would you set the character encoding for your page?
+
+## --answers--
+
+A `meta` element in the `body`.
+
+### --feedback--
+
+Metadata is behind-the-scenes information.
```

Comment:
not sure if this hint is super helpful for a beginner.

maybe the hint could be updated to mention that there are to main sections in the html document (head and body) and which one would be best to contain meta data like character encoding?

Reply:
I changed it to `Character encoding is metadata information.` What do you think?

Reply:
works for me